modules/tinymce/src/core/main/ts/table/TableSize.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { TableSize } from '@ephox/snooker';
+import { SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
```

Comment:
This import path needs fixing (it's also incorrect in a few other files).

- [ ] Fix core import paths for moved files

Reply:
Have fixed the import paths for a few of the files